soc/nxp/imxrt/imxrt7xx/Kconfig

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_MIMXRT798S_CM33_CPU0
+	select CPU_CORTEX_M33
+	select CLOCK_CONTROL
+	select CPU_CORTEX_M_HAS_DWT
+	select ARM
+	select CPU_HAS_ARM_SAU
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select ARM_TRUSTZONE_M
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select HAS_MCUX
+	select HAS_MCUX_SYSCON
+	select HAS_MCUX_CACHE
+	select HAS_GLIKEY
+	select SOC_RESET_HOOK
+
+config SOC_MIMXRT798S_CM33_CPU1
+	select CPU_CORTEX_M33
+	select CLOCK_CONTROL
+	select CPU_CORTEX_M_HAS_DWT
+	select ARM
+	select CPU_HAS_ARM_SAU
+	select CPU_HAS_ARM_MPU
+	select CPU_HAS_FPU
+	select ARMV8_M_DSP
+	select ARM_TRUSTZONE_M
+	select CPU_CORTEX_M_HAS_SYSTICK
+	select HAS_MCUX
+	select HAS_MCUX_SYSCON
+	select HAS_GLIKEY
+	select SOC_RESET_HOOK
```

Comment:
Will CPU core 1 need the reset hook to run? Just asking since the secondary core would only be booted by the primary, right?

Reply:
Yes, I have removed it for cm33_cpu1.